types/express-serve-static-core/index.d.ts

```diff
@@ -30,8 +30,12 @@ import { EventEmitter } from "events";
 import { Options as RangeParserOptions, Result as RangeParserResult, Ranges as RangeParserRanges } from "range-parser";
 
 export interface NextFunction {
-    // tslint:disable-next-line callable-types (In ts2.1 it thinks the type alias has no call signatures)
-    (err?: any): void;
+    (err?: any): void; // tslint:disable-line callable-types (In ts2.1 it thinks the type alias has no call signatures)
+    /**
+     * "Break-out" of a router by calling {next('router')};
+     * @see {https://expressjs.com/en/guide/using-middleware.html#middleware.router}
+     */
+    (deferToNext: "router"): void; // tslint:disable-line callable-types (In ts2.1 it thinks the type alias has no call signatures)
```

Comment:
I'm pretty sure the checks are now startin from 2.8, so that lint is no longer required here to siable tslint `callbacl-types` rule